### PR TITLE
Fix dot in cross-compiling

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by h-vetinari** [comment]
Thanks a lot! 🙏

I suggest we fold this into #298, which is basically complete and ready for merge (modulo this issue).

**Comment by mgorny** [comment]
I'll test and merge it into #298.

**Comment by h-vetinari** [comment]
The failures here
```
/Users/runner/miniforge3/conda-bld/libtorch_1734438514677/work/conda_build.sh: line 217: wheel: command not found
```
are likely fixed by https://github.com/conda-forge/pip-feedstock/pull/133

**Comment by h-vetinari** [comment]
OK, this got picked into #298, but still fails for osx + generic there (though not for osx +MKL... 🤔; this is using an older MKL because intel stopped supporting osx-64). Additionally, linux + MKL failed as well (rest still running). None of these are cross-compiled, which is AFAIU why the upstream [logic](https://github.com/pytorch/pytorch/blob/v2.5.1/cmake/Modules/FindBLAS.cmake#L320-L384) for these variables never triggers, as it's guarded by `IF (CMAKE_CROSSCOMPILING)`.

**Comment by hmaarrfk** [comment]
is it ok if we decouple it from the CUDA stuff in #298? Its pretty hard to keep both in mind, and they are very distinct ideas.

I think given the challenges of this repo, it is perfectly fine to attack the problems one at a time. In the debugging phase, which you seem to suggest we still are in, I think we can add skips to reduce the redundant load on the CIs.

I also think letting Isuru work on his own branch helps empower him to move faster instead of coming into conflicts with a larger effort.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -106,6 +106,7 @@ requirements:
     # other requirements
     - python 3.12  # [megabuild]
     - python       # [not megabuild]
+    - wheel
```
Comment (h-vetinari) [inline recipe/meta.yaml:109]: not necessary; it was already fixed by the pip PR I mentioned

**Comment by h-vetinari** [comment]
> is it ok if we decouple it from the CUDA stuff in #298? Its pretty hard to keep both in mind, and they are very distinct ideas.

Fine by me. I only advocated its inclusion because it seemed like a done deal... I've started a debugging job in #299 that adds two more commits on top to #298. If that doesn't yield anything, we can back out that commit and just skip the tests.

**Comment by h-vetinari** [comment]
Since this PR does not include the test suite where this happens, we won't actually see whether this changes anything (which is why I based the run in #299 on #298). I'd suggest to cancel the run on the GPU server here, to not waste resources.

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml:110]: It's better to be explicit about the dependencies we use.

**Comment by hmaarrfk** [comment]
this unfortunately needs to get rebased, and somebody needs to validate that this does work.

**Comment by isuruf** [comment]
Merged in #298

**Pull request closed without merging**